includes/class-wc-payments.php

```diff
@@ -418,11 +422,28 @@ public static function add_plugin_links( $links ) {
 	 */
 	public static function register_gateway( $gateways ) {
 		$gateways[] = self::$card_gateway;
+
+		if ( false === WC_Payments_Features::is_grouped_settings_enabled() ) {
+			return $gateways;
+		}
+
```

Comment:
```suggestion
```

Reply:
Sure, with the changes from https://github.com/Automattic/woocommerce-payments/pull/1608 now it makes sense 👍